### ISSUE TITLE
[Functional- Bot Framework Web Chat - Suggested-actions]: Incorrect functionality performing after selecting results for Suggested-actions

### ISSUE THREAD

**Issue by manojs5** [microsoft/BotFramework-WebChat#5365]
### Is it an issue related to Adaptive Cards?

No.

### User Experience:
User will get confused with the functionality of the control is incorrect and cannot access the control efficiently if the control is not behaving according to the functionality of the control.

### Note:
User credentials should NOT be included in the bug.

### What browsers and screen readers do this issue affect?

Windows: Edge with Windows Narrator

### Are there any code-based customization done to Web Chat?

No, I am using Web Chat without any customizations except "styleOptions".

### What version of Web Chat are you using?

Latest production

### Which area does this issue affect?

Suggested action

### What is the public URL for the website?

https://compulim.github.io/webchat-loader/

### How to reproduce the issue?

1. Open URL: https://compulim.github.io/webchat-loader/ and sign in using valid credentials.
2. Navigate to web chat page and select 4.18.1-main.20240927.ea7a875 (30/9/2024) version and then select mockBot link under present label and then select open web chat in new window link at bottom.
3. Navigate to Fluent UI experience chat page and type Suggested-actions word in type your message text box.
4. Navigate to the search results and navigate to each suggestions results after selecting it.
5. Verify the issue.

### What do you expect?

Correct functionality should be performing, after selecting results for Suggested-actions.

### What actually happened?

Incorrect functionality performing after selecting results for Suggested-actions
### Observarion:
After selecting ''IM back as string' showing results as Unknown command: postback imback-string. Type help to learn more.

### Do you have any screenshots or recordings to repro the issue?

![Image](https://github.com/user-attachments/assets/9f45462b-3701-490f-b770-169ef25fc6c4)
https://github.com/user-attachments/assets/7dc09dec-8bd1-4cda-92d0-60fb3ce4e598

**Comment by manojs5** [comment]
#A11ySeV1;#Accessibility;#Benchmark;#BotFrameworkWebChat-Oct24;#HCL;#Functional;#Win11-Edge(Chromium);
